Documentation/dev-guide/api_reference_v3.md

```diff
@@ -616,6 +616,7 @@ Empty field.
 | value | value is the value, in bytes, to associate with the key in the key-value store. | bytes |
 | lease | lease is the lease ID to associate with the key in the key-value store. A lease value of 0 indicates no lease. | int64 |
 | prev_kv | If prev_kv is set, etcd gets the previous key-value pair before changing it. The previous key-value pair will be returned in the put response. | bool |
+| ignore_value | If ignore_value is set, etcd updates the key using its current value. Returns an error if the key does not exist. | bool |
```

Comment:
We want to make this flag independent of lease part in case later we add lease-specific flag; see https://github.com/coreos/etcd/pull/6894#discussion_r90542077.